File: Networking/src/main/java/response/UpdatedEventsResponse.java
package response;

import DTO.EventDTO;

import java.util.List;

public class UpdatedEventsResponse implements UpdateResponse{
    private final List<EventDTO> eventsDTO;

    public UpdatedEventsResponse(List<EventDTO> eventsDTO) {
        this.eventsDTO = eventsDTO;
    }

    public List<EventDTO> getEventsDTO() {
        return eventsDTO;
    }
}

File: Networking/Response/UpdatedEventsResponse.cs
using System.Text.Json.Serialization;
using mpp_proiect_csharp_DianaGliga11.Model.DTO;

namespace Networking.Response
{
    public class UpdatedEventsResponse : UpdateResponse
    {
        [JsonPropertyName("events")]
        public List<EventDTO> Events{get;set;}

        [JsonConstructor]
        public UpdatedEventsResponse()
        {
        }

        public UpdatedEventsResponse(List<EventDTO> events)
        {
            this.Events = events;
        }
    }
}